### PR TITLE
lint startSNP

### PR DESCRIPTION
**Pull request by zkamvar**
Minor changes to syntax. Should not be merged until @smanel is okay with the changes.

### CODE REVIEW

File: use/StartSNP.Rmd [hunk 1]
```diff
@@ -97,18 +97,18 @@ We observed that heterozygosity varies among loci.
 We observed no difference between expected and observed heterozygosity.
 
 
-Basic statistics with hierfstat. Populations are states. The function `basicstat`
+Basic statistics with *hierfstat*. Populations are states. The function `basic.stats()`
 provides the observed heterozygosity ($H_o$), mean gene diversities within
-population ($H_s$), $F_{is}$, and $F_{st}$ The function `boot.ppfis` provides
-confidence interval for $F_{is}$ The fuction `indpca`  does an PCA on the
-centered matrix of individual's allele frequencies.
+population ($H_s$), $F_{is}$, and $F_{st}$ The function `boot.ppfis()` provides
```
Comment (hlapp) [inline use/StartSNP.Rmd]: Is there some punctuation (full stop, for example) missing for this and the following line.

Comment (smanel) [inline use/StartSNP.Rmd]: I confirm that full stops are missing. Do I need to add them? Where?

Comment (zkamvar) [inline use/StartSNP.Rmd]: @smanel, since the pull request is from me, it's my responsibility to add the full stops. It will be done momentarily.

**Comment by zkamvar** [comment]
@smanel, Are you okay with the changes I made?

**Pull request merged**